cfme/intelligence/reports/menus.py

```diff
@@ -180,6 +180,34 @@ def manage_folder(self, group, folder=None):
             view.manager.commit()
             view.save_button.click()
 
+    def move_reports(self, group, folder, subfolder, reports):
+        """ Moves a list of reports to a given menu
+        Args:
+            group: User group
+            folder: Parent of the subfolder where reports are to be moved.
+            subfolder: Subfolder under which the reports are to be moved.
+            reports: List of reports that are to be moved.
+        """
+        if not isinstance(reports, list):
+            reports = [reports]
+
+        with self.manage_subfolder(group, folder, subfolder) as selected_menu:
+            selected_options = selected_menu.parent_view.report_select.all_options
+            for report in reports:
+                if report in selected_options:
+                    raise Exception(
```

Comment:
Q:
why not just pass? its user wants to `a`, `b`, `c` report and `b` already in the menu. Then a better idea to just move `a` and `c` and for `b` in iteration just pass.